pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -304,7 +304,8 @@
     "Microsoft.Windows.Compatibility": {
       "StableVersions": [
         "2.0.0",
-        "2.0.1"
+        "2.0.1",
+        "2.1.0"
```

Comment:
@wtgodbe I guess this has to be done until we want to produce a stable build.

Reply:
For 2.1/2.2 we add the package to `packageindex.json` _after_ we ship it as stable - the ProdCon build infrastructure has a mechanism to mark all currently-building packages as stable, and we use this file in 2.x to mark what's already shipped stable. So if we're building 2.1.0 for this upcoming release, we should hold off on including it here until afterwards

Reply:
Ok, deleted it!